php-pre-challenge1/index.php

```diff
@@ -0,0 +1,13 @@
+<?php
+for ($i = 1; $i <= 100; $i++) {
+  if ($i % 3 == 0 && $i % 5 == 0) {
```

Comment:
```suggestion
  if ($i % 15 === 0) {
```
こう書いた方が計算量を減らせます。
サーバサイドスクリプトはフロントエンドと違い
100万アクセスとかさばくこともあるので、極力計算量を減らすべきです。